tests/Tokenizer/Transformer/TransformersTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\tests\Tokenizer\Transformer;
+
+use PhpCsFixer\Tokenizer\Transformers;
+
+/**
+ * @internal
+ *
+ * @author SpacePossum
+ */
+final class TransformersTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @param \ReflectionClass $transformerRef
+     *
+     * @dataProvider provideReflectedTransformers
+     */
+    public function testTransformersAreFinal(\ReflectionClass $transformerRef)
```

Comment:
please provide class name instead of `ReflectionClass`

example from other test what would be displayed with RC:

```
245) PhpCsFixer\Tests\ProjectCodeTest::testThatSrcClassesNotExposeProperties with data set #243 (ReflectionClass Object (...))
```

so, it's impossible to see which class looking at provided case